### ISSUE TITLE
Specialization doesn't work with dustjs 2.6+

### ISSUE THREAD

**Issue by aredridel** [krakenjs/engine-munger#24]
```
# when specialization/internationalization is enabled for dust engine with cache
not ok 39 should be equal
  ---
    operator: equal
    expected:
      null
    actual:
      {"jse_shortmsg":"Problem rendering dust template named spcl/hyde","jse_summary":"Problem rendering dust template named spcl/hyde: wtf","jse_cause":{},"message":"Problem rendering dust template named spcl/hyde: wtf"}
    at: Stub.callback (/Users/ariastewart/Projects/engine-munger/node_modules/adaro/lib/engine.js:137:22)
  ...
not ok 40 should be equal
  ---
    operator: equal
    expected: "<h1>Hello from hyde</h1>"
    actual:
    at: Stub.callback (/Users/ariastewart/Projects/engine-munger/node_modules/adaro/lib/engine.js:137:22)
  ...```

**Comment by aredridel** [comment]
Caused by https://github.com/krakenjs/engine-munger/blob/master/lib/munger.js#L83

**Comment by aredridel** [comment]
Fixed by #30
